packages/tasks/src/model-libraries.ts

```diff
@@ -295,6 +295,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/fudan-generative-vision/hallo",
 		countDownloads: `path:"hallo/net.pth"`,
 	},
+	hezar: {
+		prettyLabel: "Hezar",
+		repoName: "Hezar",
+		repoUrl: "https://github.com/hezarai/hezar",
+		docsUrl: "https://hezarai.github.io/hezar",
+		countDownloads: `path:"model.pt" OR path:"model_config.yaml" OR path:"embedding/embedding_config.yaml"`,
```

Comment:
```suggestion
		path:"model_config.yaml"`,
```

For what I can see, all models have a `model_config.yaml` file. I suppose this file is always downloaded when a user loads a model, right?

The reason I ask is because it's usually better to track downloads on a single file. If loading a model always means downloading the .pt and the model_config.yaml files, then we would be counting downloads twice.

Reply:
Done🤗. Changed it to only account for config downloads. The reason to have both `model_config` and `embedding_config` is that we also have word embedding models that do not really count as models so there's no model weights or anything there.